src/containers/Tenant/Diagnostics/Compute/Compute.js

```diff
@@ -50,7 +49,17 @@ class Compute extends React.Component {
     }
 
     componentDidUpdate(prevProps) {
-        const {autorefresh, getNodes, tenantName} = this.props;
+        const {autorefresh, getNodes, setDataWasNotLoaded, tenantName} = this.props;
+
+        if (tenantName !== prevProps.tenantName) {
+            setDataWasNotLoaded();
+            getNodes(tenantName);
```

Comment:
now this is ok
but the code is duplicated, maybe separate it into some functions here, as you did in Network?